docs/src/components/Navbar/Links.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Hide, TextLink, Stack, Button, Popover } from "@kiwicom/orbit-components";
+import { MenuHamburger } from "@kiwicom/orbit-components/lib/icons";
+
+import PuzzleIcon from "../icons/Puzzle";
+import ColorBrushIcon from "../../images/streamline-light/color-brush.svg";
+import ReadArt from "../../images/streamline-light/read-art.svg";
+import IrisScanIcon from "../../images/streamline-light/iris-scan.svg";
+import ScriptIcon from "../../images/streamline-light/script.svg";
+
+type Link = "Components" | "Guides" | "Tokens" | "Accessibility" | "Foundation";
+
+interface Props {
+  links?: Link[];
+}
+
+const LinkIcon = ({
+  name,
+  isFirst,
+  isDesktop,
+}: {
+  name?: string;
+  isFirst?: boolean;
+  isDesktop?: boolean;
+}) => {
+  if (!isDesktop) {
+    switch (name) {
+      case "Components":
+        return <PuzzleIcon stroke="#252A31" width="20px" height="20px" />;
+      case "Guides":
+        return <ReadArt stroke="#252A31" width="20px" height="20px" />;
+      case "Tokens":
+        return <ColorBrushIcon stroke="#252A31" width="20px" height="20px" />;
+      case "Accessibility":
+        return <IrisScanIcon stroke="#252A31" width="20px" height="20px" />;
+      default:
+        return <ScriptIcon stroke="#252A31" width="20px" height="20px" />;
+    }
+  }
+
+  return <>{isFirst ? null : <span>&sdot;</span>}</>;
+};
+
+const LinkList = ({ isDesktop, links }: { isDesktop?: boolean; links: Props["links"] }) => (
+  <>
+    {links &&
+      links.map((link, idx) => (
+        <Stack inline align="center">
+          <LinkIcon name={link} isFirst={idx === 0} isDesktop={isDesktop} />
```

Comment:
```suggestion
         {isDesktop ? idx === 0 ? null : <span>&sdot;</span> : <LinkIcon name={link} />}
```
And then remove the need to have `isDesktop` and `isFirst` props on `LinkIcon`. The problem here is that apparently we have an EsLint rule to disallow nested ternary. I don't necessarily disagree with this rule, but I also don't necessarily agree with it 😅
Simply because, in this situation, I think the code is more readable even with the nested ternary than having the icon/dot logic on the LinkIcon component.
WDYT?

Reply:
think it's ok, except that you suggest to use double ternary

Reply:
```{isDesktop && idx >= 0 ? <span>&sdot;</span> : <LinkIcon name={link} />}```

should be better ;)

Reply:
Good approach you took 👍